packages/png-js/rollup.config.js

```diff
@@ -15,81 +19,71 @@ const esm = {
 const getCJS = override => Object.assign({}, cjs, override);
 const getESM = override => Object.assign({}, esm, override);
 
-const configBase = {
-  input: 'src/index.js',
-  external: ['zlib'],
-  plugins: [
-    babel({
-      babelrc: false,
-      babelHelpers: 'runtime',
-      exclude: 'node_modules/**',
-      presets: [
-        [
-          '@babel/preset-env',
-          {
-            loose: true,
-            modules: false,
-            targets: { node: '12', browsers: 'last 2 versions' },
-          },
-        ],
-      ],
-      plugins: [['@babel/plugin-transform-runtime', { version: '^7.16.4' }]],
-    }),
-  ],
-};
+const input = 'src/index.js';
 
-const serverConfig = Object.assign({}, configBase, {
-  output: [
-    getESM({ file: 'lib/png-js.es.js' }),
-    getCJS({ file: 'lib/png-js.cjs.js' }),
-  ],
-  plugins: configBase.plugins.concat(
-    replace({
-      preventAssignment: true,
-      values: {
-        BROWSER: JSON.stringify(false),
+const babelConfig = ({ browser }) => ({
+  babelrc: false,
+  babelHelpers: 'runtime',
+  exclude: 'node_modules/**',
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        loose: true,
+        modules: false,
+        targets: { node: '12', browsers: 'last 2 versions' },
       },
-    }),
-  ),
-  external: ['fs'],
+    ],
+  ],
+  plugins: [['@babel/plugin-transform-runtime', { version: '^7.16.4' }]],
```

Comment:
Can we use `babelrc: true` as config so we don't duplicate config here?

Reply:
Done, I went ahead and did this for all packages.